samples/led/main.c

```diff
@@ -1,19 +1,13 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
-
     // LED pattern is:
-    //   Red:   0xC = 1100 (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
-    //   Green: 0x0 = 0000 (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
+    //   Red:   (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
+    //   Green: (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
     // This means the red LED will blink slowly and the green LED is off
-    HalWriteSMBusValue(0x20, SMC_REG_LEDSEQ, FALSE, 0xC0);
+    XSetManualLED(LED_RED, LED_RED, LED_OFF, LED_OFF);
```

Comment:
nit: There should be a comment that `XSetAutoLED` can reset the LED to the kernel-controlled mode.